src/components/DepositWidget/Styled.tsx

```diff
@@ -1,28 +1,17 @@
 import styled from 'styled-components'
 
-export const DynamicWrapper = styled.div<{ responsive: boolean }>`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-
-  ${(props): string | false =>
-    props.responsive &&
-    `
-      position: fixed;
-      top: 0;
-      left: 0;
-      bottom: 0;
-      right: 0;
-      background: #000000b5;
-  `}
-`
-
 export const InnerWrapper = styled.div`
   position: relative;
-  background-color: #f7f0ff;
+  background-color: var(--color-background-actionCards);
   border-bottom: 2px solid #0000000f;
-  border-radius: var(--border-radius);
-  width: 96%;
+  border-radius: 0 0 var(--border-radius) var(--border-radius);
+  box-shadow: var(--box-shadow);
+  margin-top: -14px;
```

Comment:
As this value is so that the component is flush with the row above, it would make more sense to use the same units (`margin-top: -0.3rem;`) as in RowTokenDiv